db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'rubygems'
+require 'nokogiri'
+require 'open-uri'
+
+page = Nokogiri::HTML(open("http://www.en365.ru/top1000b.htm"))
+rows = page.css('td#middle tr')
+
+rows.each do |row|
+  cols = row.css('td').map{|td| td.text}
+  puts cols.join(', ') # для демонстрации
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.